### PR TITLE
Added example for Sematext Logs

### PR DESCRIPTION
**Pull request by adnanrahic**
*What was added?*

- [x] Example for Sematext Logs

*Description of changes:*
This PR adds an example for using FireLens to send logs to Sematext Logs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by otisg** [comment]
@PettitWesley could you please review when you get a chance?

File: examples/fluent-bit/sematext/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "URI": "/<LOGS_TOKEN>",
+        "Port": "443",
+        "TLS": "on",
+        "Format": "json",
+        "compress": "gzip"
+    }
+},
+```
+
+> Note: If you are using the EU region of Sematext, use this Host value: `"Host": "logs-ecs-receiver.eu.sematext.com"`
```
Comment (PettitWesley) [inline examples/fluent-bit/sematext/README.md]: Can you make this bold instead of a block quote? It will make it more visible

**Review by PettitWesley — APPROVED** [review]
Thanks!

Please fix my one formatting comment and then I will merge it

**Comment by adnanrahic** [comment]
Thanks for the review @PettitWesley! I've fixed the formatting. :ok_hand:

**Pull request merged**